docs/source/_static/versionwarnings.js

```diff
@@ -0,0 +1,134 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
It seems that `s` is included in file name.

```diff
-versionwarnings.js
+versionwarning.js
```

Reply:
Oh sorry for that, great catch. I corrected the file name and the src path.